src/views/News/index.jsx

```diff
@@ -317,12 +343,50 @@ const StudentNews = () => {
       content = <GordonLoader />;
     } else {
       content = (
-        <NewsList
-          news={news}
-          personalUnapprovedNews={personalUnapprovedNews}
-          handleNewsItemEdit={handleNewsItemEdit}
-          handleNewsItemDelete={handleNewsItemDelete}
-        />
+        <>
+          <TabContext value={tabValue}>
+            <Tabs value={tabValue} onChange={handleSwitchTab} aria-label="basic tabs example">
```

Comment:
Please give this a meaningful `aria-label` for accessibility, such as `"News Lists"`